src/react/components/pages/editorPage/editorSideBar.test.tsx

```diff
@@ -13,7 +14,7 @@ describe("Editor SideBar", () => {
 
     it("Component renders correctly", () => {
         const props: IEditorSideBarProps = {
```

Comment:
I see that props is the same in many places, could we extract it to the before section?